cypress/e2e/tests/pages/manager/node-templates.spec.ts

```diff
@@ -0,0 +1,147 @@
+import ClusterManagerListPagePo from '@/cypress/e2e/po/pages/cluster-manager/cluster-manager-list.po';
+import NodeTemplatesPagePo from '@/cypress/e2e/po/pages/cluster-manager/node-templates.po';
+import ClusterManagerCreateRke1Amazonec2PagePo from '@/cypress/e2e/po/edit/provisioning.cattle.io.cluster/create/cluster-create-rke1-amazonec2.po';
+import EmberPromptRemove from '@/cypress/e2e/po/components/ember/ember-prompt-remove.po';
+
+// will only run this in jenkins pipeline where cloud credentials are stored
+describe('Node Templates', { testIsolation: 'off', tags: ['@manager', '@jenkins', '@adminUser'] }, () => {
```

Comment:
comment only, would you be able to share (privately) where jenkins results are published?

Reply:
yes i can share that with you.